### ISSUE TITLE
Clean up try/catch blocks for preferences.

### ISSUE THREAD

**Issue by wolfbeast** [MoonchildProductions/UXP#991]
With #988 implementing defaults for preference getters, we should evaluate all instances of try/catch blocks for preferences since with a default value this no longer throws, and provide defaults where prudent.
Important: just blindly getting rid of these blocks is **not good** because in a good number of cases an "unset pref" should have different behavior and there is no "default value" to use.

Depends on #988 being implemented, so filing as `on hold`.

**Comment by Ascrod** [comment]
I guess since I worked on the other two related bugs, I should take this one as well. :P

I found the following on BMO which may be relevant. Haven't analyzed them in detail just yet.
https://bugzilla.mozilla.org/show_bug.cgi?id=1344711
https://hg.mozilla.org/mozilla-central/rev/bff53385b494
https://hg.mozilla.org/mozilla-central/rev/9c92165fef20
https://hg.mozilla.org/mozilla-central/rev/340e7f8a7229

**Comment by Feodor2** [comment]
What about https://github.com/MoonchildProductions/UXP/blob/41731a7f31a1724e74b32eb7be87de2719f977c3/toolkit/mozapps/extensions/AddonManager.jsm#L232
getComplexValue did not touched, but getbool and getchar?

**Comment by Feodor2** [comment]
And https://github.com/MoonchildProductions/UXP/blob/41731a7f31a1724e74b32eb7be87de2719f977c3/toolkit/mozapps/extensions/LightweightThemeManager.jsm#L447

**Comment by mattatobin** [comment]
Leave it.
